src/ATen/native/xpu/sycl/PowKernels.cpp

```diff
@@ -36,13 +36,23 @@ static inline c10::complex<T> pow_(c10::complex<T> base, c10::complex<T> exp) {
 
 } // namespace impl
 
+#ifdef _MSC_VER
```

Comment:
Please file an issue to record the ut failure https://github.com/intel/torch-xpu-ops/issues. And comment with the issue link and with some words to explain it is a Window specific issue.

Reply:
Opened https://github.com/intel/torch-xpu-ops/issues/842